charts/argo-cd/templates/argocd-application-controller/networkpolicy.yaml

```
@@ -1,4 +1,4 @@
-{{- if .Values.global.networkPolicy.create }}
+{{- if (hasKey .Values.controller.networkPolicy "create" | ternary .Values.controller.networkPolicy.create .Values.global.networkPolicy.create)}}
```

If we go with [this](https://github.com/argoproj/argo-helm/pull/3382/files#r2203583618), maybe like this?
```suggestion
{{- if or .Values.controller.networkPolicy.create .Values.global.networkPolicy.create }}
```

---

resolved by commit